app/src/main/java/org/groundplatform/android/ui/datacollection/tasks/AbstractTaskFragment.kt

```diff
@@ -231,13 +226,7 @@ abstract class AbstractTaskFragment<T : AbstractTaskViewModel> : AbstractFragmen
   /** Adds the action buttons to the UI. */
   private fun renderButtons() {
     taskView.actionButtonsContainer.composeView.setComposableContent {
-      Row(
-        horizontalArrangement = Arrangement.SpaceBetween,
-        modifier = Modifier.fillMaxWidth().padding(8.dp),
-      ) {
-        // TODO: Previous button should always be positioned to the left of the screen.
-        //  Rest buttons should be aligned to the right side of the screen.
-        // Issue URL: https://github.com/google/ground-android/issues/2417
```

Comment:
Why have we removed these lines?

Reply:
this issue is already closed - https://github.com/google/ground-android/issues/2417